src/sample.env

```diff
@@ -0,0 +1,59 @@
+# A lot of the settings reflect what knex already exposes, with slightly altered names
```

Comment:
why you have `#` before every value? disregarding required or optional?

Reply:
if they want to use the value they can uncomment, if they want to make their own, they will still be able to see the example. that is the idea